pkg/platform/model/projects.go

```diff
@@ -120,13 +120,8 @@ func DefaultBranchForProject(pj *mono_models.Project) (*mono_models.Branch, *fai
 	return nil, FailNoDefaultBranch.New(locale.T("err_no_default_branch"))
 }
 
-// CreateProject will create the project on the platform
-func CreateProject(owner, name, hostPlatform string, lang *language.Supported, langVersion string, private bool) (*mono_models.Project, strfmt.UUID, *failures.Failure) {
-	_, fail := CreateEmptyProject(owner, name, private)
-	if fail != nil {
-		return nil, "", fail
-	}
-
+// InitializeProject will create the project on the platform
+func InitializeProject(hostPlatform string, lang *language.Supported, langVersion string) (strfmt.UUID, *failures.Failure) {
```

Comment:
This function should be dropped now. The requirement stuff can live in the init function. CommitInitial is what should be called from init.